spec/v2/index.bs

```diff
@@ -1699,6 +1626,160 @@ For each new ProductFootprint, the [=data owner=] MUST
+
+Organizations (especially organizations initiatives) are encouraged to define the relevant `$entity-idtype$` for products within their industry separately. 
+
+## Examples ## {#product-identifier-examples}
+
+Below is a list of examples of <{ProductFootprint/productIds}> as used in the <{ProductFootprint}> datat type clarifying the use of well-known and custom URN namespaces for identifying products. 
+
+<figure id="product-id-examples">
+<table class="data">
+<thead>
+  <tr>
+    <td>Product ID type
+    <td>Example
+<tbody>
+  <tr>
+    <td>
+    Company-specific
+    
+    Identifiers using the `pact` namespace, created by a given company for the purposes of uniquely identifying their products
+
+    <td><div class=example>
+    ```json
+    ["urn:pact:sample.com:product-id:44055-9c05bc35-68f8"]
+    ```</div>
+    <div class=example>
+    ```json
+    ["urn:pact:sample-buyer.com:buyer-id:103403453"]
+    ```</div>
+    <div class=example>
+    ```json
+    ["urn:pact:sample-supplier.com:supplier-id:1234"]
+    ```</div>
+  <tr>
+    <td>ISBN
+
+    Well known ISBN standard, see [iana.org](https://www.iana.org/assignments/urn-formal/isbn)
+    <td><div class=example>
+    ```json
+    ["urn:isbn:978-951-0-18435-6"]
+    ```
+  <tr>
+    <td>GTIN (widely used)
+
+    GTIN is not an official IANA registered namespace, however in practice it is used to specify GTINs as a URN. See [gs1.org](https://www.gs1.org/standards/id-keys/gtin)
+    <td><div class=example>
+    ```json
+    ["urn:gtin:4712345060507"]
+    ```
+  <tr>
+    <td>UUID
+
+    Globally Unique Identifiers. See [[RFC9562]]
+    <td><div class=example>
+    ```json
+    ["urn:uuid:69585GB6-56T9-6958-E526-6FDGZJHU1326"]
+    ```
+  <tr>
+    <td>CAS Registry Number
+
+    Unique identification number assigned to every chemical substance described in the open scientific literature See [cas.org](https://www.cas.org/cas-data/cas-registry)
+    <td><div class=example>
+    ```json
+    ["urn:pact:cas.org:substance-id13463-67-7"]
+    ```
+  <tr>
+    <td>InChI (International Chemical Identifier)
+
+    InChI is a standard identifier for chemical databases that facilitates effective information management across chemistry. See [inchi-trust.org](https://www.inchi-trust.org/)
+    <td><div class=example>
+    ```json
+    ["urn:pact:inchi-trust.org:substance-id:$INCHI-ID$"]
+    ```
+  <tr>
+    <td>Combined example of a substance
+
+    Combined example of a substance (Titan Dioxide of supplier Sigmaaldrich)
+    <td><div class=example>
+    ```json
+    ["urn:pact:sigmaaldrich.com:supplier-id:14021",
+    "urn:pact:cas.org:substance-number:13463-67-7",
+    "urn:pact:iupac.org:substance-name:dioxotitanium",
+    "urn:pact:inchi-trust.org:substance-id:1S,/2O.Ti",
+    "urn:pact:inchi-trust.org:substance-key:
+    GWEVSGVZZGPLCZ-UHFFFAOYSA-N"]
+    ```
```

Comment:
I wonder if in the example above, we are also mixing product ids from product classification ids? I recall Bjoern wrote this example but that was before we decided to split productIDs and product classification ids. Maybe we should check with him to pick from this list just the set of unique product ids?

Reply:
Agreed, let's check